charts/invenio/templates/install-init-job.yaml

```diff
@@ -56,10 +56,24 @@ spec:
             secretKeyRef:
               name: invenio-secrets
               key: INVENIO_CSRF_SECRET_SALT
+        volumeMounts:
+          {{- if .Values.invenio.vocabularies }}
+          {{- range $key, $value := .Values.invenio.vocabularies }}
+          - name: vocabularies
+            mountPath: "/opt/invenio/var/instance/app_data/vocabularies/{{ $key }}"
+            subPath: "{{ $key }}"
+            {{- end }}
+          {{- end }}
         {{- if .Values.web.resources }}
         resources: {{- toYaml .Values.web.resources | nindent 10 }}
         {{- end }}
       restartPolicy: OnFailure
+      volumes:
+        {{- if .Values.invenio.vocabularies}}
```

Comment:
If `.Values.invenio.vocabularies` is null, `volumes` will be empty. Better to not include `volumes` if not used?